packages/ui-components/components/Circle/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function Circle({
+  color = 'bg-gray-80',
+  size = 'w-1 h-1',
```

Comment:
size is not consistent among all components

Reply:
The Circle component is only a component used in Shuttleflow progress. we can delete it here.